### PR TITLE
fix whitespaces in application descriptions

### PR DESCRIPTION
**Pull request by milesstoetzner**
see https://github.com/OpenTOSCA/ui/pull/82

### CODE REVIEW

File: servicetemplates/http%3A%2F%2Fopentosca.org%2Fexample%2Fapplications%2Fservicetemplates/MyTinyToDo-MySql_Docker-w1/SELFSERVICE-Metadata/data.xml [hunk 1]
```diff
@@ -1,13 +1,9 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<selfservice:Application
-        xmlns:selfservice="http://www.eclipse.org/winery/model/selfservice"
->
+<selfservice:Application xmlns="http://docs.oasis-open.org/tosca/ns/2011/12" xmlns:winery="http://www.opentosca.org/winery/extensions/tosca/2013/02/12" xmlns:yml="http://docs.oasis-open.org/tosca/ns/simple/yaml/1.3" xmlns:selfservice="http://www.eclipse.org/winery/model/selfservice" xmlns:testwineryopentoscaorg="http://test.winery.opentosca.org">
```
Comment (wederbn) [inline servicetemplates/http%3A%2F%2Fopentosca.org%2Fexample%2Fapplications%2Fservicetemplates/MyTinyToDo-MySql_Docker-w1/SELFSERVICE-Metadata/data.xml:2]: Why additional namespace declarations? Are they set by Winery? I would remove them as they are not used within the document

File: servicetemplates/http%3A%2F%2Fopentosca.org%2Fexample%2Fapplications%2Fservicetemplates/PetClinic_MySQL-OpenStack-w1/SELFSERVICE-Metadata/data.xml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<selfservice:Application xmlns="http://docs.oasis-open.org/tosca/ns/2011/12" xmlns:winery="http://www.opentosca.org/winery/extensions/tosca/2013/02/12" xmlns:selfservice="http://www.eclipse.org/winery/model/selfservice" xmlns:testwineryopentoscaorg="http://test.winery.opentosca.org">
+<selfservice:Application xmlns="http://docs.oasis-open.org/tosca/ns/2011/12" xmlns:winery="http://www.opentosca.org/winery/extensions/tosca/2013/02/12" xmlns:yml="http://docs.oasis-open.org/tosca/ns/simple/yaml/1.3" xmlns:selfservice="http://www.eclipse.org/winery/model/selfservice" xmlns:testwineryopentoscaorg="http://test.winery.opentosca.org">
```
Comment (wederbn) [inline servicetemplates/http%3A%2F%2Fopentosca.org%2Fexample%2Fapplications%2Fservicetemplates/PetClinic_MySQL-OpenStack-w1/SELFSERVICE-Metadata/data.xml:2]: See above

File: servicetemplates/http%3A%2F%2Fopentosca.org%2Fexample%2Fapplications%2Fservicetemplates/RealWorld-Application_Angular-Spring-w1/SELFSERVICE-Metadata/data.xml [hunk 3]
```diff
@@ -1,13 +1,9 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<selfservice:Application
-        xmlns:selfservice="http://www.eclipse.org/winery/model/selfservice"
->
+<selfservice:Application xmlns="http://docs.oasis-open.org/tosca/ns/2011/12" xmlns:winery="http://www.opentosca.org/winery/extensions/tosca/2013/02/12" xmlns:yml="http://docs.oasis-open.org/tosca/ns/simple/yaml/1.3" xmlns:selfservice="http://www.eclipse.org/winery/model/selfservice" xmlns:testwineryopentoscaorg="http://test.winery.opentosca.org">
```
Comment (wederbn) [inline servicetemplates/http%3A%2F%2Fopentosca.org%2Fexample%2Fapplications%2Fservicetemplates/RealWorld-Application_Angular-Spring-w1/SELFSERVICE-Metadata/data.xml:2]: See above

File: servicetemplates/http%3A%2F%2Fopentosca.org%2Fexample%2Fapplications%2Fservicetemplates/MyTinyToDo-MySql_Docker-w1/SELFSERVICE-Metadata/data.xml (hunk 1 above)
Comment (milesstoetzner) [reply servicetemplates/http%3A%2F%2Fopentosca.org%2Fexample%2Fapplications%2Fservicetemplates/MyTinyToDo-MySql_Docker-w1/SELFSERVICE-Metadata/data.xml:2]: Yes they are set by winery.

I used the Management UI to adjust the text.
Shall I get rid of these?

Applies to the other comments too.

Comment (milesstoetzner) [reply servicetemplates/http%3A%2F%2Fopentosca.org%2Fexample%2Fapplications%2Fservicetemplates/MyTinyToDo-MySql_Docker-w1/SELFSERVICE-Metadata/data.xml]: done

File: servicetemplates/http%3A%2F%2Fopentosca.org%2Fexample%2Fapplications%2Fservicetemplates/RealWorld-Application_Angular-Spring-w1/SELFSERVICE-Metadata/data.xml (hunk 3 above)
Comment (milesstoetzner) [reply servicetemplates/http%3A%2F%2Fopentosca.org%2Fexample%2Fapplications%2Fservicetemplates/RealWorld-Application_Angular-Spring-w1/SELFSERVICE-Metadata/data.xml]: done

**Review by wederbn — APPROVED** [review]
LGTM

**Pull request merged**